checker/src/main/java/org/checkerframework/checker/pico/jdk.astub

```diff
@@ -0,0 +1,355 @@
+import org.checkerframework.checker.pico.qual.Mutable;
```

Comment:
Now that we're moving PICO into EISOP, we should add these annotations directly into eisop/jdk.
If you use the same branch name, hopefully CI picks up the right branch.